src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1204,6 +1208,17 @@ static Library()
                 BuiltinFunctionsCore.Pi,
                 NoErrorHandling(Pi)
             },
+            {
+                BuiltinFunctionsCore.PlainText,
+                StandardErrorHandling<FormulaValue>(
+                    BuiltinFunctionsCore.PlainText.Name,
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: NoOpAlreadyHandledByIR,
+                    checkRuntimeTypes: DeferRuntimeTypeChecking,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
```

Comment:
>ReturnBehavior.AlwaysEvaluateAndReturnResult [](http://example.com/codeflow?start=36&length=44)

maybe `ReturnBehavior.ReturnBlankIfAnyArgIsBlank ` ?

Reply:
Looks like ReturnBlankIfAnyArgIsBlank has been used for functions will multiple args and make sure return blank if any of them blank. But this plaintext function has only 1 input. and logic of it return blank if input blank already.

Reply:
Correct, I think the same should work for 1 Arg as well.

Reply:
Updated here.